### PR TITLE
[global] Changing the range of ports that listen to nodes

### PR DESCRIPTION
**Pull request by tarabrind**
## Description
Moving most ports that are listened to on nodes to the range 4200-4299.

## Why do we need it, and what problem does it solve?
This will make it easier for administrators to create network policies on firewalls within the corporate infrastructure. They will be able to add a single range to the policy.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: network
type: chore
summary: Moved most ports that are listened to on nodes to the range 4200-4299.
impact: |
  The following system pods will restart:
  * node-local-dns,
  * cloud-provider-*,
  * runtime-audit-engine,
  * metallb,
  * cilium-agent,
  * kube-proxy,
  * registry-packages-proxy,
  * bashible-apiserver,
  * capi-controller-manager,
  * machine-controller-manager,
  * network-policy-engine,
  * ingress-nginx with HostPortWithFailover inlet,
  * runtime-audit-engine.
  Note that you will need to change the access policies on the firewalls before upgrading the cluster.
impact_level: high
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by apolovov** [comment]
/e2e/abort vsphere;Standard;containerd;1.25 9414769969 9414769969-1-con-1-25 /sys/deckhouse-oss/install:pr8598 ubuntu@10.220.203.75

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've recognized your '/e2e/abort' command and started the workflow...

File: modules/050-network-policy-engine/templates/daemonset.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ spec:
       maxAllowed:
         cpu: 40m
         memory: 100Mi
+    {{- include "helm_lib_vpa_kube_rbac_proxy_resources" . | nindent 4 }}
```
Comment (tarabrind) [inline modules/050-network-policy-engine/templates/daemonset.yaml:29]:
```suggestion
```

File: docs/documentation/_data/deckhouse-ports.yml [hunk 2]
```diff
@@ -9,16 +9,21 @@ groups:
     description:
       en: etcd replication
       ru: Репликация etcd
-  - ports: "9443"
```
Comment (apolovov) [inline docs/documentation/_data/deckhouse-ports.yml:12]: Need a note on the rendered page about the range.

**Review by apolovov — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by apolovov** [comment]
/e2e/abort static;Static;containerd;1.25 9676359312 9676359312-1-con-1-25 /sys/deckhouse-oss/install:pr8598  - for abort failed cluster

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by apolovov** [comment]
/e2e/abort static;Static;containerd;1.25 9676359312 9676359312-1-con-1-25 /sys/deckhouse-oss/install:pr8598

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by apolovov** [comment]
/e2e/abort vsphere;Standard;containerd;1.25 9678346650 9678346650-1-con-1-25 /sys/deckhouse-oss/install:pr8598

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/openstack'...

**Comment by apolovov** [comment]
/e2e/abort vsphere;Standard;containerd;1.25 9678549584 9678549584-1-con-1-25 /sys/deckhouse-oss/install:pr8598

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/vsphere'...

**Review by miklezzzz — APPROVED** [review]

**Review by vladimirGuryanov — APPROVED** [review]

File: testing/matrix/linter/rules/rules.go [hunk 3]
```diff
@@ -564,6 +564,31 @@ func objectSecurityContext(object storage.StoreObject) errors.LintRuleError {
...
+	}
+
+	if o.Unstructured.GetKind() == "DaemonSet" && o.Unstructured.GetName() == "node-local-dns" &&
+		o.Unstructured.GetNamespace() == "kube-system" && c.Name == "coredns" {
+		if (hostNetworkUsed && p.ContainerPort == 53) || p.HostPort == 53 {
+			return true
+		}
+	}
+
+	return false
```
Comment (nabokihms) [inline testing/matrix/linter/rules/rules.go:589]: Do we need to skip this controller? If so, it must be explained in the code via comments.

Comment (tarabrind) [reply testing/matrix/linter/rules/rules.go]: NTP port was moved to the range and other rules were commented out.

File: docs/documentation/pages/NETWORK_SECURITY_SETUP_RU.md [hunk 4]
```diff
@@ -9,6 +9,6 @@ lang: ru
 * Включен режим туннелирования трафика между подами ([настройки](modules/021-cni-cilium/configuration.html#parameters-tunnelmode) для CNI Cilium, [настройки](modules/035-cni-flannel/configuration.html#parameters-podnetworkmode) для CNI Flannel).
 * В случае необходимости интеграции с внешними системами (например, LDAP, SMTP или прочие внешние API), с ними разрешено сетевое взаимодействие.
 * Локальное сетевое взаимодействие полностью разрешено в рамках каждого отдельно взятого узла кластера.
-* Разрешено взаимодействие между узлами по портам, приведенным в таблицах на текущей странице.
+* Разрешено взаимодействие между узлами по портам, приведенным в таблицах на текущей странице. Обратите внимание, что большинство портов входит в диапазон 4200-4299. При добавлении новых компонентов платформы, им будут назначаться порты из этого диапазона (при наличии возможности).
```
Comment (z9r5) [inline docs/documentation/pages/NETWORK_SECURITY_SETUP_RU.md:12]:
```suggestion
* Разрешено взаимодействие между узлами по портам, приведенным в таблицах на текущей странице. Обратите внимание, что большинство портов входит в диапазон 4200-4299. При добавлении новых компонентов платформы им будут назначаться порты из этого диапазона (при наличии возможности).
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @qw1mb0. I've started the workflow for label 'deploy/web/stage'...

**Comment by z9r5** [comment]
/e2e/abort static;Static;containerd;1.25 9677923650 9677923650-1-con-1-25 /sys/deckhouse-oss/install:pr8598 astra@192.168.199.144

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by z9r5** [comment]
@tarabrind Please resolve the conflict

**Review by z9r5 — APPROVED** [review]

**Comment by z9r5** [comment]
@tarabrind Please fix the DCO test

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by apolovov — APPROVED** [review]

**Comment by tarabrind** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.27 9758914045 9758914045-1-con-1-27 /sys/deckhouse-oss/install:pr8598 redos@158.160.123.51

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by apolovov** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.27 9760592076 9760592076-1-con-1-27 /sys/deckhouse-oss/install:pr8598 redos@84.201.157.155

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by apolovov** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.27 9760592548 9760592548-1-con-1-27 /sys/deckhouse-oss/install:pr8598 centos@18.153.229.181

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by tarabrind** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.27 9758914045 9758914045-1-con-1-27 /sys/deckhouse-oss/install:pr8598 redos@158.160.123.51

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @tarabrind. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/vsphere'...

**Pull request merged**